cmd/list_test.go

```diff
@@ -0,0 +1,491 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package cmd
+
+import (
+	"net/http"
+	"reanahub/reana-client-go/utils"
+	"testing"
+
+	"github.com/go-gota/gota/series"
+
+	"golang.org/x/exp/slices"
+)
+
+var listServerPath = "/api/workflows"
+
+func TestList(t *testing.T) {
+	successResponse := `{
+		"total": 2,
+		"items": [
+			{
+				"created": "2022-07-28T12:04:37",
+			  	"id": "my_workflow_id",
```

Comment:
Cosmetics: some of these lines are indented using tabs only, some a combination of tabs and spaces.  E.g. the "created" line and "id" line start on the same column, yet use different indentation. Might be good to harmonise.